pkg/controller/clusterpool/collections.go

```diff
@@ -263,6 +263,19 @@ func isBroken(cd *hivev1.ClusterDeployment, pool *hivev1.ClusterPool, logger log
 		logger.Infof("Cluster %s is broken due to resume timeout", cd.Name)
 		return true
 	}
+	if cd.Spec.ClusterPoolRef != nil && cd.Spec.ClusterPoolRef.ClusterDeploymentCustomizationRef != nil {
```

Comment:
The way the logic is laid out, this needs to go above the `Check for resume timeout` block. Otherwise we could short out on any of the above `return false`s.